client/js/table-view.js

```diff
@@ -18,11 +18,13 @@ class TableView {
   	this.headerRowEle = document.querySelector('THEAD TR');
     this.bodyEle = document.querySelector('TBODY');
     this.formulaBarEle = document.querySelector('#formula-bar');
+    this.footEle = document.querySelector('TFOOT');
   }
 
   renderTable() {
   	this.renderTableHeader();
```

Comment:
initDomReferences() {
    this.headerRowEl = document.querySelector('THEAD TR');
    this.bodyEl = document.querySelector('TBODY');
    this.formulaBarEl = document.querySelector('#formula-bar');
    this.footEl = document.querySelector('TFOOT');
  }